weblate/checks/chars.py

```diff
@@ -290,6 +296,28 @@ def check_single(self, source, target, unit):
         return self.check_chars(source, target, -1, ("!", "！", "՜", "᥄", "႟", "߹"))
 
 
+class EndInterrobangCheck(TargetCheck):
+    """Check for final interrobang expression."""
+
+    check_id = "end_Interrobang"
+    name = gettext_lazy("Mismatched interrobang")
+    description = gettext_lazy(
+        "Source and translation do not both end with an interrobang expression"
+    )
+
+    def check_single(self, source, target, unit):
+        if not source or not target:
+            return False
+
+        interrobang_sets = [("!?", "?!"), ("？！", "！？")]
+
+        for sets in interrobang_sets:
+            if (source.endswith(sets)) != (target.endswith(sets)):
+                return True
```

Comment:
Why are full width and standard width separated here? I think it is expected to use full width in full width languages like Chinese and standard width in Latin languages. So having source in English and target in Chinese, the interrobang width is expected to differ.

Reply:
I thought every interrobang expression could be used in any language, and it was supposed to ensure with the code that both source and translation belonged to the same set. I fixed it to allow full width and standard width to be used interchangeably. I hope it's correct now!